packages/fake-driver/test/unit/driver.spec.js

```diff
@@ -38,4 +38,21 @@ describe('FakeDriver', function () {
     await d1.deleteSession(session1Id);
     await d2.deleteSession(session2Id);
   });
+  it('should return deprecated commands called', async function () {
```

Comment:
I'm not sure if this test is actually useful. It does not verify any of the actual base driver behavior